CHANGELOG.md

```diff
@@ -1,7 +1,32 @@
 # Parse-SDK-JS
 
 ### master
-[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.19.0...master)
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/3.0.0...master)
+
+## 3.0.0
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.19.0...3.0.0)
+
+**Breaking Changes**
+For security purposes, logIn will default to `POST` instead of `GET` method.
```

Comment:
Is there a PR reference for this or has this been a sec adv?

Reply:
I'll add it

Reply:
I like that the changelog explains the breaking change in some detail. We should aim for that in any changelog.